etp-backend/src/main/clj/solita/etp/service/valvonta_kaytto/email.clj

```diff
@@ -170,7 +196,10 @@
     (send-email! valvonta toimenpide osapuoli [document] templates-omistaja)))
 
 (defn send-toimenpide-email! [db aws-s3-client valvonta toimenpide osapuolet]
-  (let [postinumero (maybe/map* #(luokittelu-service/find-luokka
+  (let [tiedoksi (-> (valvonta-kaytto-db/select-template db {:id (:template-id toimenpide)})
```

Comment:
Tämä olisi ehkä ollut kivempi hakea ulkopuolella ja passata generate-pdf-documentille ja tälle sieltä. En kuitenkaan tehnyt näin, kun viimeksi mainittu ottaa jo valmiiksi paljon parametreja eikä tuntunut sopivalta liittää tätä esim toimenpide-parametriin lisäattribuutiksi.